translations/traditionalchinese/minor-activities.ts

```diff
@@ -0,0 +1,62 @@
+import {Translations} from "../../server/chat";
```

Comment:
```suggestion
import type {Translations} from "../../server/chat";
```

(and so on for all the others)